bindings/java/Swig/post-build.cmd

```diff
@@ -30,7 +30,7 @@ if "%is_gpu%" == "true" (
   )
 )
 
-for %%x in (libiomp5md.dll mklml.dll mkldnn.dll Cntk.Math-%version%.dll Cntk.PerformanceProfiler-%version%.dll Cntk.Core-%version%.dll Cntk.Core.JavaBinding-%version%.dll) do (
+for %%x in (opencv_world310.dll libiomp5md.dll mklml.dll mkldnn.dll Cntk.Math-%version%.dll Cntk.PerformanceProfiler-%version%.dll Cntk.Core-%version%.dll Cntk.Core.JavaBinding-%version%.dll) do (
```

Comment:
Are you planning to distribute jar for Debug as well? If yes, opencv_world310**d**.dll should be used for Debug builds.

Reply:
I think we only plan on releasing release jars.

Reply:
In that case, you can either stop generating Jar or make sure to copy the correct dlls for debug builds .
Currently, debug jar will contain zip.dll and zlib.dll which will be useless without opencv_world310d.dll.